### DISCUSSION TITLE
My personal advice on mining strategy in epoch3

### DISCUSSION THREAD

**Discussion by kevingzhang** [tearust/teaproject#67] [Show and tell]
I wrote this https://github.com/tearust/teaproject/discussions/31#discussioncomment-1308088

Please go take a look. I bet this will help you in the epoch3 contest. If you finally made it, buy me a coffee, ... or no, a tea.

**Comment by geopor** [comment]
- Being early to claim the lowest staking slots of others mining CML, and staking immature CML into other mining machines as you wait for them to mature are still valuable strategies. But they will not count for as much as before since RA rewards have been slashed. Make sure to regularly come back to the TApp page to see if there are more TApps your CML can host as your mining machines mature in productivity.

- Keep an eye on non-productive assets. This could be having a too high COFFEE or TEA balance as well as frozen seeds in your wallet that could be deposited for Genesis loans.

- Strategically pay back COFFEE debt. Selling to COFFEE when the rate is in favor of TEA is one way to pay back your debt earlier than everyone else rushing to do so at the end.

- Watch the YouTube videos representing each TApp before deciding which to invest in. You'll want to buy the tokens of the ones that you think have potential to gain many views over the course of the contest.
